src/core/map.js

```diff
@@ -64,6 +69,12 @@ geo.map = function (arg) {
     return new geo.map(arg);
   }
   arg = arg || {};
+
+  if (arg.node === undefined || arg.node === null) {
+    console.warn('map creation requires a node');
```

Comment:
In most of geojs we issue warnings and not errors.  We can change it if you like (in which case, perhaps we should revisit the other warnings, too).